lib/browser/chrome-devtools.js

```diff
@@ -71,6 +71,7 @@ ipcMainUtils.handle('ELECTRON_INSPECTOR_CONTEXT_MENU', function (event, items, i
   const window = event.sender.getOwnerBrowserWindow()
 
   menu.once('menu-will-close', () => {
+    // menu-will-close is emitted before the click handler, but needs to be sent after
```

Comment:
why does it need to be sent after? what happens if it isn't?

Reply:
added more details